src/Command/MigrateCommand.php

```diff
@@ -80,6 +80,7 @@ public function buildOptionParser(ConsoleOptionParser $parser): ConsoleOptionPar
             'help' => "Mark any migrations selected as run, but don't actually execute them",
             'boolean' => true,
         ])->addOption('dry-run', [
+            'short' => 'x',
```

Comment:
Was this change intentional?

Reply:
Yes. I missed this short alias and the tests for CompletionShell integration identified the regression.